railties/lib/rails/generators/rails/app/templates/Gemfile

```diff
@@ -34,7 +34,7 @@ group :development, :test do
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
   <%- if depends_on_system_test? -%>
   # Adds support for Capybara system testing and selenium driver
-  gem 'capybara', '~> 2.13.0'
+  gem 'capybara', '~> 2.13'
```

Comment:
@matthewd Capybara follows semver except for features that are designated beta.  The beta features are opt-in and the user is notified they are beta, and may change, when they turn them on.